VisualStudio/Options/SkuOption.cs

```diff
@@ -6,13 +6,19 @@ namespace VisualStudio
 {
     class SkuOption : OptionSet<Sku?>
     {
-        readonly static string[] shortcuts = new[] { "e", "ent", "enterprise", "p", "pro", "professional", "c", "com", "community" };
+        readonly static string[] shortcuts = new[]
+        {
+            "e", "ent", "enterprise",
+            "p", "pro", "professional",
+            "c", "com", "community",
+            "b", "bld", "buildtools",
```

Comment:
I'm not in love with the `bld` tla :). But I'd add `build` too, and `test` for the other. Both are short enough that the three-letter one might be unnecessary, I think...